about.html

```diff
@@ -0,0 +1,115 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Africa Agriculture Summit|2023</title>
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Icons+Outlined" />
+    <link rel="stylesheet" href="./css/style.css" />
+    <script src="./js/index.js" defer></script>
+  </head>
+  <body>
+    <header>
+      <div class="desktop-nav">
+        <div class="face-twitt">
+          <img src="./images/facebook.png" alt="facebook" class="facebook">
+          <img src="./images/twitter.png" alt="twitter" class="twitter">
+        </div>
+        <ul>
+          <li><a href="#">English</a></li>
+          <li><a href="#">Home</a></li>
+          <li><a href="#">Logout</a></li>
+        </ul>
+      </div>
+    </div>
+        <img src="./images/menu.png" class="menu" alt="menu-icon">
+        <div class="desktop-items">
+          <a href="index.html"><img src="./images/logo.png" alt="" class="desktop-logo"></a>
+          <ul>
+            <li><a href="about.html" class="about">About</a></li>
+            <li><a href="#">Program</a></li>
+            <li><a href="#">Sponsors</a></li>
+            <li><a href="#">News</a></li>
+            <li><a href="#" class="campaign">AA Campaign</a></li>
+          </ul>
+        </div>
+      <nav class="hidden">
+        <img src="./images/cancel.png" alt="" class="cancel">
+        <ul>
+          <li><a href="#">Home</a></li>
+          <li><a href="about.html">About</a></li>
+          <li><a href="#">Program</a></li>
+          <li><a href="#">Feature</a></li>
+        </ul>
+      </nav>
+      <main>
+            
+        <h1 class=" about-title">Africa Agriculture Summit 2023</h1>
+
+      </main>
```

Comment:
- [ ] Dear @oluwatobi-fasasi, I think you have missed the small line of text on top of the main heading (`Africa Agriculture Summit 2023` ) Please add a small line of text as shown on the Original design. This is to make sure we have something identical to the original design.